CONTRIBUTING.md

```diff
@@ -0,0 +1,169 @@
+# Contribute to Trinity
+
+This document describes how to contribute to Trinity.
+
+We encourage everyone with knowledge of IOTA technology to contribute.
+
+Thanks! :heart:
+
+<details>
+<summary>Do you have a question :question:</summary>
+<br>
+
+If you have a general or technical question, you can use one of the following resources instead of submitting an issue:
+
+- [**Developer documentation:**](https://docs.iota.org/) For official information about developing with IOTA technology
+- [**Discord:**](https://discord.iota.org/) For real-time chats with the developers and community members
+- [**IOTA cafe:**](https://iota.cafe/) For technical discussions with the Research and Development Department at the IOTA Foundation
+- [**StackExchange:**](https://iota.stackexchange.com/) For technical and troubleshooting questions
+</details>
+
+<br>
+
+<details>
+<summary>Ways to contribute :mag:</summary>
+<br>
+
+To contribute to Trinity on GitHub, you can:
+
+- Report a bug
+- Suggest a new feature
+- Build a new feature
+- Contribute to the documentation
+
+</details>
+
+<br>
+
+<details>
+<summary>Report a bug :bug:</summary>
+<br>
+
+This section guides you through reporting a bug. Following these guidelines helps maintainers and the community understand the bug, reproduce the behavior, and find related bugs.
+
+### Before reporting a bug
+
+Please check the following list:
+
+- **Do not open a GitHub issue for [security vulnerabilities](SECURITY.MD)**, instead, please contact us at [security@iota.org](mailto:security@iota.org).
+
+- **Ensure the bug was not already reported** by searching on GitHub under [**Issues**](https://github.com/iotaledger/trinity/issues). If the bug has already been reported **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
+
+**Note:** If you find a **Closed** issue that seems similar to what you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+### Submitting A Bug Report
+
+To report a bug, [open a new issue](https://github.com/iotaledger/trinity/issues/new), and be sure to include as many details as possible, using the template.
```

Comment:
```suggestion
To report a bug, [open a new issue](https://github.com/iotaledger/trinity-wallet/issues/new?labels=T+-+Bug&template=bug_report.md&title=), and be sure to include as many details as possible, using the template.
```
This will direct readers to the appropriate template